joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/testfixtures/RubyCode2CpgFixture.scala

```diff
@@ -12,7 +12,7 @@ trait RubyFrontend extends LanguageFrontend {
   override val fileSuffix: String = ".rb"
 
   override def execute(sourceCodeFile: File): Cpg = {
-    implicit val defaultConfig: Config = Config()
+    implicit val defaultConfig: Config = Config(true)
```

Comment:
Rather, make this configurable and not true by default for all test suites. May require adding a `def enableDependencyDownload` or changing `RubyFrontend` to an abstract class or something

Reply:
done.